### PR TITLE
Adding icon star legend on team view footer

### PR DESCRIPTION
**Pull request by juuh42dias**
rgsoc teams app 6

### CODE REVIEW

File: app/views/teams/_table.html.slim [hunk 1]
```diff
@@ -37,4 +37,5 @@ table.table.table-striped.table-bordered.table-condensed
         td = render "supervisor/shared/performance_icon", status: team.performance.evaluation.to_s
 
 p.hint
-  | <i class="icon-star"></i> Sponsored team
+  | <span class="icon-star-sponsored"></span>Sponsored team &nbsp&nbsp
```
Comment (carpodaster) [inline app/views/teams/_table.html.slim]: The HTML entities need to be terminated by an `;`

File: app/views/teams_info/index.html.slim [hunk 2]
```diff
@@ -24,4 +24,5 @@ table.table.table-striped.table-bordered.table-condensed
         td = team.post_info
 
 p.hint
-  | <i class="icon-star"></i> Sponsored team
+  | <span class="icon-star-sponsored"></span>Sponsored team &nbsp&nbsp
```
Comment (carpodaster) [inline app/views/teams_info/index.html.slim]: The HTML entities need to be terminated by an `;`

**Comment by juuh42dias** [comment]
@carpodaster , done :)

**Comment by carpodaster** [comment]
Yay, thanks!

**Pull request merged**